Module.php

```diff
@@ -311,4 +312,18 @@ protected function corePanels()
             'timeline' => ['class' => 'yii\debug\panels\TimelinePanel']
         ];
     }
+
+    /**
+     * @inheritdoc
+     * @since 2.0.7
+     */
+    protected function defaultVersion()
+    {
+        $packageInfo = Json::decode(file_get_contents(__DIR__ . DIRECTORY_SEPARATOR . 'composer.json'));
+        $extensionName = $packageInfo['name'];
```

Comment:
do we really need to read this from the composer.json file? The name is hardly ever going to change so it can just be hard coded here.